tests/integration_tests/clients/bedrock/test_chat_completions.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from ai21 import AI21BedrockClient, AsyncAI21BedrockClient, BedrockModelID
+from ai21.models._pydantic_compatibility import _to_dict
+from ai21.models.chat import ChatMessage
+
+_SYSTEM_MSG = "You're a support engineer in a SaaS company"
+_MESSAGES = [
+    ChatMessage(content=_SYSTEM_MSG, role="system"),
+    ChatMessage(content="Hello, I need help with a signup process.", role="user"),
+]
+
+
+def test_chat_completions__when_stream__last_chunk_should_hold_bedrock_metrics():
+    client = AI21BedrockClient()
+    response = client.chat.completions.create(
+        messages=_MESSAGES,
+        model=BedrockModelID.JAMBA_INSTRUCT_V1,
+        stream=True,
+    )
+
+    last_chunk = None
+    for chunk in response:
+        assert chunk.id is not None
+        assert chunk.choices is not None
+        last_chunk = chunk
```

Comment:
if you only case about the last chunk. You can do something like this maybe - 

`last_chunk = list(response)[-1]`

And remove the loop